TappyReader/Tappy.swift

```diff
@@ -23,8 +23,8 @@
 
 import Foundation
 
-protocol Tappy {
-    func setResponseListener(listener : @escaping (TCMPMessage) -> ())
+@objc protocol Tappy {
+    func setResponseListener(listener : @escaping (TCMPMessage, String) -> ())
```

Comment:
I can't accept this change as this is confusing to other developers that would wonder why it is being passed both as a proper TCMPMessage and as a string.  

If you want this String version of a response listener in the core SDK, my recommendation  is to add another type of response listener , call it say, `responseListenerJSON' that would have the following method signature:

`func responseListenerJSON(jsonResponse : String)`

 then add a `setResponseListenerJSON` and a `removeResponseListenerJSON`.

Then in `SerialTappy`, you could add a call to such  a JSON response listener here: https://github.com/TapTrack/TCMPTappy-iOS/blob/4a99d205cbf85263c26f0d14665102f719076e78/TappyReader/SerialTappy.swift#L64